presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcPageFileRewriter.java

```diff
@@ -59,7 +59,6 @@
 import static java.util.stream.Collectors.toMap;
 
 public final class OrcPageFileRewriter
```

Comment:
Just call this `OrcFileRewriter`

Reply:
Thanks for the review. Class has been renamed to `OrcFileRewriter`.